thirdparty/sdcv/CMakeLists.txt

```diff
@@ -29,7 +25,7 @@ set(CFG_CMD sh -c "${SOURCE_DIR}/configure ${CFG_OPTS}")
 ko_write_gitclone_script(
     GIT_CLONE_SCRIPT_FILENAME
     git://github.com/koreader/sdcv.git
-    d1b85743ded5e99232aed59e84129a551bfc8c82
+    c23763affc89c6355550e37a1bcb8a1aa848c24d
```

Comment:
The way I merged it changed it to `4455cf31bb4a424370e29d858dc07cf8d8f5de04`. Sorry about that.